sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -21,12 +21,14 @@ class MissingClientIdError extends Error {
 
 enum GapiScopes {
   BIGQUERY,
```

Comment:
Could we do without needing to list BigQuery and GCS separately?

Reply:
Removed BIGQUERY and GCS scopes.